jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/deploy/DeployServerlessApplicationPanel.kt

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.services.lambda.deploy
+
+import com.intellij.execution.util.EnvVariablesTable
+import com.intellij.execution.util.EnvironmentVariable
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.project.Project
+import com.intellij.ui.AnActionButton
+import com.intellij.ui.IdeBorderFactory
+import com.intellij.ui.SimpleListCellRenderer
+import com.intellij.ui.ToolbarDecorator
+import com.intellij.ui.components.panels.Wrapper
+import software.aws.toolkits.jetbrains.services.cloudformation.Parameter
+import software.aws.toolkits.jetbrains.services.ecr.resources.EcrResources.LIST_REPOS
+import software.aws.toolkits.jetbrains.services.ecr.resources.Repository
+import software.aws.toolkits.jetbrains.services.s3.resources.S3Resources.listBucketNamesByActiveRegion
+import software.aws.toolkits.jetbrains.ui.ResourceSelector
+import software.aws.toolkits.resources.message
+import javax.swing.JButton
+import javax.swing.JCheckBox
+import javax.swing.JLabel
+import javax.swing.JPanel
+import javax.swing.JRadioButton
+import javax.swing.JTextField
+
+class DeployServerlessApplicationPanel(private val project: Project) {
+    lateinit var newStackName: JTextField
+    lateinit var createS3BucketButton: JButton
```

Comment:
add private sets

Reply:
Added (and was able to set a few more `private`)